### ISSUE TITLE
coref_tab rule description is inconsistent?

### ISSUE THREAD

**Issue by ftyers** [amir-zeldes/xrenner#53]
In the docs here: https://corpling.uis.georgetown.edu/xrenner/doc/models.html#coref-rules-tab it says:
```
The file coref_rules.tab lists a cascade of coreference matching rules of the form:

    ANA;ANT;DIR;DIST;PROP
```

e.g. there are 5 ';' separated columns. But in the rules the 'DIR' column appears to be left out.

```
#first match identical proper markables
form="proper";form="proper"&text=$1;100;nopropagate
#identify coreferent light heads based on identical modifiers
lemma=/^one$/;mod=$1&anytext&form!="proper"&!sameparent;4;nopropagate
#salvage unmatched pronouns - cataphoric cases like "in her speech, the chairwoman..."
text=/^(his|her|its)$/;form!=/pronoun/&func=/nsubj/&lookahead;0;propagate_invert
```
and:
```
$ cat coref_rules.tab | grep '[^;]\+;[^;]\+;[^;]\+;[^;]\+;' | wc -l
0
```

does 'DIR' have a default value ?

**Comment by amir-zeldes** [comment]
Yes, you're right, the documentation is out of date. The default DIR is backwards, and it is now specified as an option inside the 2nd column (so there are only 4 columns), like this:

&lookahead

The example lower in the documentation is actually correct, just the text was not updated correctly.

**Comment by amir-zeldes** [comment]
Fixed in 6068069d82ee4bf13154d56b606de5e5d1ee24f9
